ckanext/ontario_theme/templates/internal/snippets/search_form.html

```diff
@@ -69,13 +69,13 @@ <h2 aria-label='{{ hidden_searchbar_title }}'><p class="theme_p" aria-hidden="tr
               {% set choices = h.scheming_field_choices(h.scheming_field_by_name(h.scheming_get_organization_schema('organization')['fields'], 'jurisdiction'))%}
             {% endif %}
 
-            {{ h.snippet('snippets/translate_facets.html', fields=schema_dict, name=item_id, item=item, scheming_choices=choices, extras=extras) }}
+            {{ h.snippet('snippets/translate_facets.html', fields=schema_dict, name=item_id, item=item, scheming_choices=choices) }}
 
               {% if item.active %}
                 <span class="filtered pill">
                   <span class="pill-contents">
                     {{item.display_name}}
-                    <a href="{{ h.remove_url_param(item_id, item.name, extras=extras, alternative_url=alternative_url) }}" class="remove" title="{{ _('Remove') }}"><i class="fa fa-times"></i></a>
+                    <a href="{{ facets.remove_field(field, item.name) }}" class="remove" title="{{ _('Remove') }}"><i class="fa fa-times"></i></a>
```

Comment:
also here why are `extras` removed?

Reply:
This was the line that was causing the error. I removed the h.remove_url_param and changed it for facets.remove_field. This was the original code, and that's why it is working on stage and not on test.